### ISSUE TITLE
[Request] SAM: Replace Hissatsu: Guren with Zanshin

### ISSUE THREAD

**Issue by Propagandist1** [MKhayle/XIVComboExpanded#450]
**Describe the feature you'd like**
I'd like a simple option to replace Hissatsu: Guren with Zanshin.  

There already exists an option to replace Hissatsu: Kyuten with both Hissatsu: Guren and Zanshin.  However, I think it would be best if we could have an option to keep Hissatsu: Kyuten as its own button, but have Hissatsu: Guren and Zanshin combined.  This is because Hissatsu: Kyuten is a PBAOE, while Hissatsu: Guren and Zanshin are line/cone AOEs.  The player positioning is different for executing PBAOEs vs line/cone AOEs.

You could also have an option to replace Hissatsu: Guren with Shoha when available.  

Thank you for all of your hard work making this the best combo plugin!  I swear, one of these days I'm going to try and learn how Github works, so I can submit some code instead of just requests!

**Comment by MKhayle** [comment]
@kaedys did you get what they said because I'm not a SAM player yet (but I plan to one day)

**Comment by MKhayle** [comment]
if you don't tackle that one I'll do it

**Comment by Propagandist1** [comment]
Your mod is the main reason I can enjoy all the jobs!

**Comment by kaedys** [comment]
Ya, similar to the PCT one.  Kyuten is the standard AoE Kenki spender, the AoE equivalent of Shinten.  We have combos to place both Guren and Zanshin on it, just as we have combos to place Senei and Zanshin on Shinten.  The ask, I believe, is to make a separate combo that place Zanshin on Guren (and probably also an equivalent for Senei), since they have similar usage patterns, and in the case of Guren, similar AoE targeting.  Shouldn't be too tough to implement.

**Comment by Propagandist1** [comment]
That is correct.  A separate combo to place Zanshin on Hissatsu: Guren without involving Hissatsu: Kyuten.  This is because Guren and Zanshin are the same "type" of attack (line/cone AOE) while Kyuten is PBAOE.  Seems to me that even if you can execute Guren or Zanshin, you may still want to use Kyuten by itself because you're in the middle of a mob pack.

Senei probably doesn't need any edits, since there's already an option to combine it with Shinten.

But if you're making a separate combo to put Zanshin on Guren, it may also be valuable to have the option to also put Shoha on Guren, when available.  Priority Zanshin (if > 50 Kenki and Zanshin Ready) > Shoha (if off CD and = 3 stacks of Meditation) > Guren.

However just the ability to put Zanshin on Hissatsu: Guren when available (if > 50 Kenki and Zanshin Ready) would make me happy.  :)
